src/Simulator/FloodyFluidSimulator.h

```diff
@@ -44,6 +44,9 @@ class cFloodyFluidSimulator :
 	Returns whether the block was changed or not. */
 	bool HardenBlock(cChunk * a_Chunk, int a_RelX, int a_RelY, int a_RelZ, BLOCKTYPE a_BlockType, NIBBLETYPE a_Meta);
 
+	/** Checks if neighboring blocks are soakable, if so, converts them to soaked version. */
```

Comment:
Explain in the doxy-comment what the `a_BlockType` and `a_Meta` params are - they aren't immediately obvious.

Reply:
And done